app/controllers/application_controller.rb

```diff
@@ -13,6 +13,7 @@ class ApplicationController < ActionController::Base
   helper_method :current_role
 
   rescue_from ActiveRecord::RecordNotFound, with: :not_found!
+  rescue_from Date::Error, with: :invalid_date
```

Comment:
If we go with this approach, maybe could we rescue from a custom error for this purpose like `InvalidDateRange` or something? Because since `Date::Error` is from the standard library we might at some point raise this error in a different place and we may not want to respond in the same way.

Reply:
Yeah good point! I created the new custom error and updated the unit test [here](https://github.com/rubyforgood/human-essentials/pull/4925/commits/d06e1a703522c4715951aade83e214aef5f67397)